app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
```

Comment:
I agree, but I opted for `object` because it didn't seem right to have so many nested shapes:
```
annotation: React.PropTypes.shape({
      answers.propTypes.shape({
          // what goes in here? Type-checking all the possible answers seems crazy
      }),
      choice.propTypes.string,
      filters.propTypes.shape({
         // what goes in here?
      })
    })
```
Thoughts?